docs/examples/api-reference/aggregations/firstk.py

```diff
@@ -0,0 +1,152 @@
+from datetime import datetime
+from typing import List
+
+import pandas as pd
+import pytest
+
+from fennel.testing import mock
+
+__owner__ = "satwant@fennel.ai"
+
+
+@mock
+def test_basic(client):
+    # docsnip basic
+    from fennel.datasets import dataset, field, pipeline, Dataset, FirstK
+    from fennel.dtypes import Continuous
+    from fennel.lib import inputs
+    from fennel.connectors import source, Webhook
+
+    webhook = Webhook(name="webhook")
+
+    @source(webhook.endpoint("Transaction"), disorder="14d", cdc="append")
+    @dataset
+    class Transaction:
+        uid: int
+        amount: int
+        timestamp: datetime
+
+    @dataset(index=True)
+    class Aggregated:
+        uid: int = field(key=True)
+        amounts: List[int]
+        timestamp: datetime
+
+        @pipeline
+        @inputs(Transaction)
+        def firstk_pipeline(cls, ds: Dataset):
+            return ds.groupby("uid").aggregate(
+                # docsnip-highlight start
+                amounts=FirstK(
+                    of="amount",
+                    limit=10,
+                    dedup=False,
+                    window=Continuous("1d"),
+                    drop_nulls=False,
```

Comment:
Could you please make this a single word `dropnull` so that it reads similar to existing `dropnull` operator?